lnbits/core/templates/core/index.html

```diff
@@ -338,7 +338,7 @@ <h5 class="q-my-md">{{SITE_TAGLINE}}</h5>
           outline
           color="grey"
           type="a"
-          href="https://legend.lnbits.com/paywall/GAqKguK5S8f6w5VNjS9DfK"
+          href="https://demo.lnbits.com/paywall/GAqKguK5S8f6w5VNjS9DfK"
```

Comment:
i think this link is invalid now right? should be changed?

Reply:
done!